pkgs/default.nix

```diff
@@ -6922,32 +6922,32 @@ inherit (pkgs) mesa;};
            license = stdenv.lib.licenses.bsd3;
          }) {};
       "cardano-sl" = callPackage
-        ({ mkDerivation, aeson, ansi-terminal, ansi-wl-pprint, base, binary
-         , bytestring, canonical-json, cardano-crypto, cardano-sl-binary
-         , cardano-sl-block, cardano-sl-core, cardano-sl-crypto
-         , cardano-sl-db, cardano-sl-delegation, cardano-sl-infra
-         , cardano-sl-lrc, cardano-sl-networking, cardano-sl-ssc
-         , cardano-sl-txp, cardano-sl-update, cardano-sl-util, cborg, cereal
-         , conduit, constraints, containers, cpphs, cryptonite, data-default
-         , deepseq, directory, dns, ed25519, ekg-core, ekg-statsd, ekg-wai
-         , ether, exceptions, extra, filelock, filepath, fmt, formatting
-         , generic-arbitrary, half, hashable, hspec, kademlia, lens
-         , log-warper, lrucache, memory, mmorph, monad-control, MonadRandom
-         , mtl, neat-interpolation, network-transport, network-transport-tcp
-         , optparse-applicative, parsec, plutus-prototype, pvss, QuickCheck
-         , random, reflection, resourcet, rocksdb-haskell-ng
-         , safe-exceptions, safecopy, serokell-util, servant, servant-server
-         , servant-swagger, stdenv, stm, systemd, tagged, template-haskell
-         , text, text-format, time, time-units, transformers
-         , transformers-base, universum, unix, unliftio
-         , unordered-containers, vector, wai, warp, warp-tls, yaml
+        ({ mkDerivation, aeson, ansi-terminal, ansi-wl-pprint, async, base
+         , binary, bytestring, canonical-json, cardano-crypto
+         , cardano-sl-binary, cardano-sl-block, cardano-sl-core
+         , cardano-sl-crypto, cardano-sl-db, cardano-sl-delegation
+         , cardano-sl-infra, cardano-sl-lrc, cardano-sl-networking
+         , cardano-sl-ssc, cardano-sl-txp, cardano-sl-update
+         , cardano-sl-util, cborg, cereal, conduit, constraints, containers
+         , cpphs, cryptonite, data-default, deepseq, directory, dns, ed25519
+         , ekg-core, ekg-statsd, ekg-wai, ether, exceptions, extra, filelock
+         , filepath, fmt, formatting, generic-arbitrary, half, hashable
+         , hspec, kademlia, lens, log-warper, lrucache, memory, mmorph
+         , monad-control, MonadRandom, mtl, neat-interpolation
+         , network-transport, network-transport-tcp, optparse-applicative
+         , parsec, plutus-prototype, pvss, QuickCheck, random, reflection
+         , resourcet, rocksdb-haskell-ng, safe-exceptions, safecopy
+         , serokell-util, servant, servant-server, servant-swagger, stdenv
+         , stm, systemd, tagged, template-haskell, text, text-format, time
+         , time-units, transformers, transformers-base, universum, unix
+         , unliftio, unordered-containers, vector, wai, warp, warp-tls, yaml
```

Comment:
How about one packet per line so that the diff will be cleaner the next time we need to add an entry?

Reply:
Isn't `pkgs/default.nix` a generated file (generated by `pkgs/generate.sh`)?

Reply:
If it is generated then it shouldn't be in git.

Reply:
Indeed it is generated, by `pkgs/generate.sh`. It has to be in git in order for CI to pass. Maybe there's a better way, I don't know, but so be it for now.